tech.json

```diff
@@ -894,6 +894,17 @@
               ]
             }
           ]
+        },
+        {
+          "name": "canWaterShineCharge",
+          "requires": [
+            "canSuitlessMaridia",
+            "SpeedBooster"
+          ],
+          "note": [
+            "Starting from an air environment, run into water and charge a spark in the water. This is typically done while crossing a door transition from a room with air physics to a room with water.",
+            "This is easy to do; the tech just represents knowledge that it is possible."
```

Comment:
Techs tagged with `obscure` are like correct to be where they are now, but we'll want to revisit once some amount of time has passed for this to become more normal.